### PR TITLE
Update Node.js to latest v14.17.6 LTS

### PR DESCRIPTION
**Pull request by cyntler**
## Summary

I decided to update Node.js to the latest LTS version.
The change affects ci.json and the .nvmrc file.

## How did you test this change?

I ran testing scripts (`yarn test`, `yarn test --prod`), building script (`yarn build`) and Flow types checking (`yarn flow dom`). Looks like is working fine.

### CODE REVIEW

**Comment by bvaughn** [comment]
> I decided to update Node.js to the latest LTS version.

Why?

**Comment by cyntler** [comment]
@bvaughn Okay, so I read this article: https://www.gumlet.com/blog/whats-new-in-node-14/ - Node version 14 updates V8 Runtime to the >8.1 version. In the article we can find information about improved performance thanks to the OSR caching (https://v8.dev/blog/v8-release-79#osr-caching). In Node 14, thanks to the V8 update, we also have optional chaining.
I saw that React has version 12, I decided to update to version 14. I saw that the scripts work, tests too. React is my favorite library, I was looking for something first to change. My dream is to become a contributor and help in development. I thought it might be a good start;)

**Comment by gaearon** [comment]
Thanks! I'm not sure if the nvmrc file is used by anything, but the CI change shouldn't hurt. :)

File: .codesandbox/ci.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "packages": ["packages/react", "packages/react-dom", "packages/scheduler"],
   "buildCommand": "download-build-in-codesandbox-ci",
-  "node": "12",
+  "node": "14",
```
Comment (wab712) [inline .codesandbox/ci.json:4]: Mata.no?

**Pull request merged**